acceptance/pe/tests/basic.rb

```diff
@@ -0,0 +1,21 @@
+require 'scooter'
+
+test_name "basic validation of puppetdb-cli subcommands" do
```

Comment:
This test looks good, but I am unable to get it to pass on Windows2012r2. @ajroetker have you had success with running `puppet query` against PE with an RBAC token and a global config for the pdb-cli tools?